lib/code_climate/test_reporter/formatter.rb

```diff
@@ -98,7 +99,13 @@ def round(numeric, precision)
         Float(numeric).round(precision)
       end
 
-      private
+      def write_to_file?
+        return true if tddium? || ENV["CODECLIMATE_TO_FILE"]
+        if ENV["TO_FILE"]
+          warn "TO_FILE is deprecated, use CODECLIMATE_TO_FILE"
+          true
```

Comment:
changed

On Sat, Apr 18, 2015 at 7:30 AM, Noah Davis <notifications@github.com>
wrote:

> In lib/code_climate/test_reporter/formatter.rb
> <https://github.com/codeclimate/ruby-test-reporter/pull/84#discussion_r28644426>
> :
>
> > @@ -98,7 +99,13 @@ def round(numeric, precision)
> >          Float(numeric).round(precision)
> >        end
> >
> > -      private
> > +      def write_to_file?
> > +        return true if tddium? || ENV["CODECLIMATE_TO_FILE"]
> > +        if ENV["TO_FILE"]
> > +          warn "TO_FILE is deprecated, use CODECLIMATE_TO_FILE"
> > +          true
>
> It might be simpler / easier to read if you separate the deprecation
> warning from the return value of this method.
>
> That is this method could return, more simply without early returns the
> result of tddium? || ENV["CODECLIMATE_TO_FILE"] || ENV["TO_FILE"]. Before
> that you could print a deprecation warning.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codeclimate/ruby-test-reporter/pull/84/files#r28644426>
> .
>